src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -147,15 +143,13 @@ EmbeddedAssemblies::find_entry_in_type_map (const char *name, uint8_t map[], Typ
 inline const char*
 EmbeddedAssemblies::typemap_java_to_managed (const char *java)
 {
-#if defined (DEBUG) || !defined (ANDROID)
 	for (TypeMappingInfo *info = java_to_managed_maps; info != nullptr; info = info->next) {
 		/* log_warn (LOG_DEFAULT, "# jonp: checking file: %s!%s for type '%s'", info->source_apk, info->source_entry, java); */
 		const char *e = reinterpret_cast<const char*> (bsearch (java, info->mapping, info->num_entries, info->entry_length, TypeMappingInfo_compare_key));
 		if (e == nullptr)
 			continue;
 		return e + info->value_offset;
 	}
-#endif
```

Comment:
@grendello same here, I could skip this for-loop unless the shared runtime is used.

Reply:
ditto :)